### DISCUSSION TITLE
Time Zone

### DISCUSSION THREAD

**Discussion by kaznad** [polymind-inc/acmebot#401] [Q&A] [answered]
Hi,

How do I set the time zone?
In the readme it says to set a application settings for WEBSITE_TIME_ZONE. But what should the value be? 
Eg. for UTC-05:00, should the value be "-05:00 UTC"

**Comment by shibayan** [comment, marked as answer]
Run the `tzutil.exe /L` command to get the time zone name, as described in the `WEBSITE_TIME_ZONE` documentation.
For example, UTC-5 is `Eastern Standard Time`
https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-timer?tabs=csharp#ncrontab-time-zones

**Reply by kaznad** [reply]
Got it. Thank you.
